openassessment/conf/locale/es_ES/LC_MESSAGES/djangojs.po

```diff
@@ -589,3 +589,13 @@ msgstr "Esta tarea de respuesta abierta ya ha sido publicada. Los cambios solo a
 #: xblock/static/js/src/studio/oa_edit.js:319
 msgid "error count: "
 msgstr "recuento de errores:"
+
+#: openassessment/xblock/static/dist/openassessment-studio.979e8b88dd0d9cee68f7.js:25
+#: openassessment/xblock/static/js/src/lms/components/WaitingStepList.jsx:38
+msgid "Action"
+msgstr "Acción"
```

Comment:
[OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html) is a work in progress, but I don't think it's enough to put these translations here.. I think they need to be set in Transifex, otherwise the next transifex pull will override them.

If this happens, could you work with the #translations-wg to resolve it?

(Not a blocker for merge here, just FYI.)